packages/validator/templates/1.0.0/ois.json

```diff
@@ -0,0 +1,184 @@
+{
+  "oisFormat": {
+    "__regexp": "^(0|[1-9]\\d*)\\.(0|[1-9]\\d*)\\.(0|[1-9]\\d*)(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+  },
+  "title": {
+    "__regexp": "^[a-zA-Z0-9-_\\s]{1,64}$"
```

Comment:
spaces are allowed?

Reply:
and where does the 64 come from?

Reply:
I don't quite remember, there were many changes like this during last few months, but this just seems like something I either read in documentation or consulted with @bbenligiray